site/app/models/notebook/Notebook.php

```diff
@@ -154,29 +154,42 @@ private function getMarkdownData($cell) {
     * @param array $new_notebook a notebook config to parse
     * @param int $version which version to get notebook submission values from
     * @param string $student_id which student's notebook to pull data from
+    * @param int $display_version selected version (may not be active version)
+    * @param string $gradeable_id
     */
-    public function getMostRecentNotebookSubmissions(int $version, array $new_notebook, string $student_id): array {
+    public function getMostRecentNotebookSubmissions(int $version, array $new_notebook, string $student_id, int $display_version, string $gradeable_id): array {
```

Comment:
You've typed this as `int` here but pass in `null` below? If `null` is a valid value, you need to add a `?` before the type, like `?int`.